types/bookshelf/index.d.ts

```diff
@@ -435,6 +437,17 @@ declare namespace Bookshelf {
     }
 
     interface CollectionCreateOptions extends ModelOptions, SyncOptions, CollectionAddOptions, SaveOptions {}
+
+    interface FetchPageOptions {
```

Comment:
Should extend from FetchAllOptions: _Besides the basic options that can be passed to Model#fetchAll, there are some additional pagination options that can be specified._

https://bookshelfjs.org/api.html#Model-instance-fetchPage